README.md

```diff
@@ -21,3 +21,35 @@ A set of recipes for automating Linux tasks.
 Want me to write something to automate your Linux tasks?
 
 I would love to do so. Just open up an issue in the issue tracker.
+
+
+
+# Resources
+
+## Beginners (to learn shell scripting)
```

Comment:
Good work with the resources. I would love to see some good youtube playlist links in this section as well.

Reply:
Ok sir, will add some YouTube playlist as well and would try to change the name of the commit using terminal.